scene/resources/animation.cpp

```diff
@@ -62,6 +62,22 @@ bool Animation::_set(const StringName &p_name, const Variant &p_value) {
 			compression.pages[i].time_offset = page["time_offset"];
 		}
 		compression.enabled = true;
+	} else if (prop_name == SNAME("markers")) {
```

Comment:
```suggestion
		return true;
	} else if (prop_name == SNAME("markers")) {
```

Reply:
Sorry, I deleted it, but if the email was sent, then the review here was wrong and needs `return true`.